meteor-app/.meteor/packages

```diff
@@ -31,7 +31,6 @@ session@1.1.7
 dynamic-import@0.4.0
 babel-compiler@7.1.1
 meteortesting:mocha
-force-ssl@1.1.0
```

Comment:
You shouldn't be taking this out. It is a required package for both staging and production. This is a major mistake, and is potentially a sackable offence, as you are compromising the security of the app.
Your deployment simply needs to enable the letsencrypt option and proxy and it will work. You need to set up a dns name for it to work. AJ has done this if you need help